#include <exception>
class ExpectedError : public std::exception {};